lldb/source/Plugins/Language/Swift/SwiftLanguage.h

```diff
@@ -75,6 +75,12 @@ class SwiftLanguage : public Language {
   ConstString
   GetDemangledFunctionNameWithoutArguments(Mangled mangled) const override;
 
+  /// Returns whether two SymbolContexts correspond to funclets of the same
+  /// async function.
+  /// If either SymbolContext is not a funclets, nullopt is returned.
+  std::optional<bool>
+  AreEquivalentFunctions(const SymbolContext &sc1,
```

Comment:
The name of this function doesn't describe well what it does.
Also not great: `AsyncFuncletsOfSameFunction`?

Reply:
Where is this used?

Reply:
> The name of this function doesn't describe well what it does.
> 
> Also not great: `AsyncFuncletsOfSameFunction`?

We're renaming it on the upstream PR that introduces the base class interface (see the PR linked previously: https://github.com/llvm/llvm-project/pull/112720)

Reply:
> Where is this used?

It's used by the next commit in this PR and in the unittest of this commit